bleak/__init__.py

```diff
@@ -263,7 +277,13 @@ def apply_filter(d: BLEDevice, ad: AdvertisementData):
 
 
 class BleakClient:
-    """The Client Interface for Bleak Backend implementations to implement.
+    """The Client interface for connecting to a specific BLE GATT server and communicating with it.
+
+    A BleakClient can be used as an asynchronous context manager in which case it automatically
+    connects and disconnects.
+
+    How many BLE connections can be active simultaneously, and whether connections can be active while
+    scanning depends on the backend.
```

Comment:
These depend on the Bluetooth adapter hardware, not the backend.

Reply:
Fixed.